bats/fb-verify-packages.bats

```diff
@@ -6,15 +6,34 @@ set -o pipefail
 load os_helper
 load foreman_helper
 
-@test "verify no old SCL packages" {
+@test "verify no rh-ruby24, tfm-ror51 or tfm-ror52 SCL packages" {
   tSetOSVersion
+  if ! tIsRedHatCompatible; then
+    skip "SCL package checks only applicable on EL systems"
+  fi
+
+  FOREMAN_VERSION=$(tForemanVersion)
+  if [[ $FOREMAN_VERSION != 2.* ]]; then
+    skip "Verification of no rh-ruby24, tfm-ror51 or tfm-ror52 packages only applies to Foreman 2.*"
+  fi
+
+  if rpm --query --all | grep --quiet --extended-regexp 'rh-ruby24|tfm-ror51|tfm-ror52'; then
+    exit 1
+  fi
+}
+
+@test "verify no rh-ruby25 SCL packages in Foreman 2.5" {
+  tSetOSVersion
+  if ! tIsRedHatCompatible; then
+    skip "SCL package checks only applicable on EL systems"
+  fi
+
   FOREMAN_VERSION=$(tForemanVersion)
+  if [[ $FOREMAN_VERSION != 2.5* ]]; then
```

Comment:
I think @evgeni's comment can be addressed by such a check. It can check Foreman 2.5+
```suggestion
  if [[ $FOREMAN_VERSION != 2.[5-9]* ]]; then
```

Reply:
I can try a few conditions here.